packages/yoga/src/Dialog/web/index.js

```diff
@@ -0,0 +1,10 @@
+import Dialog from './Dialog';
+import Header from './Header';
+import Content from './Content';
+import Actions from './Actions';
+
+Dialog.Header = Header;
+Dialog.Content = Content;
+Dialog.Actions = Actions;
```

Comment:
I think that instead of Actions, we could call it Footer, to be more generic. Is this a bad idea?

Reply:
I think it's a great idea, but I followed the same model as the card's components.

[Example](https://gympass.github.io/yoga/components/card/default)


But I can follow up with your suggestion, which I liked.